app/styles/flex.scss

```diff
@@ -46,8 +19,53 @@ $align-items-map: (
   "stretch": "stretch",
 );
 
-@each $name, $property in $align-items-map {
-  .align-items-#{$name} {
-    align-items: #{$property};
+@each $prefix, $breakpoint in $breakpoint-map {
+  .#{$prefix}flex {
```

Comment:
Does this output classes such as` xsflex`, `lflex`?

Reply:
The colon is in the prefix, so the classes are s:flex, m:flex etc